install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,254 @@
+[[install-config-aggregate-logging-sizing]]
+= Aggregate Logging Sizing Guidelines 
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The xref:../install_config/aggregate_logging.adoc#overview[ElasticSearch, Fluentd and Kibana] (EFK) stack aggregates logs from nodes and applications running inside your {product-title} installation.
+Once deployed it uses link:http://www.fluentd.org/architecture[Fluentd] to aggregate event logs from all nodes, projects and pods into link:https://www.elastic.co/products/elasticsearch[Elasticsearch]. It also provides a centralized link:https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] web UI where users and administrators can create rich visualizations and dashboards with the aggregated data.
```

Comment:
Later in the docs, ES is used instead of ElasticSearch. That is fine, but let's show the user what ES stands for in the first mentioning of the term:

link:https://www.elastic.co/products/elasticsearch[ElasticSearch (ES)]